packages/opencensus-exporter-prometheus/src/prometheus-stats.ts

```diff
@@ -209,8 +220,11 @@ export class PrometheusStatsExporter implements StatsEventListener {
    * @param view View used to get the DistributionData
    * @param tags Tags used to get the DistributionData
    */
-  private getBoundaries(view: View, tags: Tags): number[] {
-    const data = view.getSnapshot(tags) as DistributionData;
+  private getBoundaries(view: View, labels: labelValues): number[] {
+    const tagValues = Object.keys(labels).map((label) => {
+      return {value: String(labels[label])};
```

Comment:
Optional: similar comment here on not needing to use a `return` if you put parens around the object